File: c-kata-8kyu/total-pressure-calculation.c
double total_pressure(double molar_mass1, double molar_mass2, double given_mass1, double given_mass2, double v, double t) {
    double M1 = (given_mass1 * 0.001) / molar_mass1;
    double M2 = (given_mass2 * 0.001) / molar_mass2;
    t = t + 273.15;
    double R = 0.082;
    return (((M1 + M2) * R * t) / v) * 1000;
}

File: kata-8kyu/total-pressure-calculation.js
solution= (molarMass1, molarMass2, givenMass1, givenMass2, volume, temp) => {
  M1 = (givenMass1 * 0.001) / molarMass1;
  M2 = (givenMass2 * 0.001) / molarMass2;
  temp = temp + 273.15;
  const R = 0.082;
  return (((M1 + M2) * R * temp) / volume) * 1000;
}
